src/objective-c/ProtoRPC/ProtoRPC.h

```diff
@@ -97,8 +97,36 @@ NS_ASSUME_NONNULL_BEGIN
 
 @end
 
+/**
+ * GRPCProtoCallFlowControllable defines the available interface methods to use when flow control
+ * is enabled. Calling these methods when flow control is disabled has no effect.
+ */
+@protocol GRPCProtoCallFlowControllable <NSObject>
```

Comment:
Probably call it GRPCControllableProtoCallFlow?

Reply:
got it. Updated the protocol naming.